src/whaleapi.py

```diff
@@ -1,19 +1,40 @@
 import json
 import requests
+import os
+import time
 
 class WhaleAPI:
-    api_key = 123
+    api_key = os.environ['API_KEY']
     url = "https://api.whale-alert.io/v1"
-    status = "status"
-    transactions = "transactions"
+    r = requests.get(url)
+    t = int(time.time())
+            
+    def errorCheck(self):
```

Comment:
```suggestion
    def error_check(self):
```

파이썬 함수와 변수는 `snake_case` 패턴을 사용합니다.